aggregator/src/aggregator/aggregation_job_creator.rs

```diff
@@ -23,11 +26,12 @@ use prio::{
     },
 };
 use rand::{random, thread_rng, Rng};
-use std::{collections::HashMap, convert::Infallible, sync::Arc, time::Duration};
+use std::{collections::HashMap, convert::Infallible, iter, sync::Arc, time::Duration};
 use tokio::{
     select,
     sync::oneshot::{self, Receiver, Sender},
     time::{self, Instant, MissedTickBehavior},
+    try_join,
```

Comment:
I can't think of a reason doing so would cause problems, but might as well be consistent -- I mostly-arbitrarily settled on using `tokio::try_join` throughout.